pkg/mount/block/blockdev.go

```diff
@@ -467,6 +508,42 @@ func (b BlockDevices) FilterName(name string) BlockDevices {
 	return partitions
 }
 
+// FilterBlockPCI returns a list of BlockDev objects whose backing
+// pci devices do not match the blocklist of PCI devices passed in.
+// FilterBlockPCI discards entries which have a matching PCI vendor
+// and device ID as an entry in the blocklist.
+func (b BlockDevices) FilterBlockPCI(blocklist pci.Devices) BlockDevices {
```

Comment:
could you test this in QEMU with virtio-blk-pci devices?

pkg/mount already has a lot of setup, it'd probably be easiest to add there in mount_integration_test.go and mount_linux_test.go

Reply:
I changed it to actually emulate an nvme device, which I think is a better test. PTAL